app/models/api_key.rb

```diff
@@ -73,9 +73,15 @@ def mfa_authorized?(otp)
   def mfa_enabled?
     return false unless user?
     return false unless user.mfa_enabled?
+    return false if initial_duration&.<(15.minutes)
     user.mfa_ui_and_api? || mfa
   end
 
+  def initial_duration
+    return unless created_at && expires_at
+    expires_at - created_at
+  end
```

Comment:
I suggest avoiding the awkward call above by making this more literal.

```suggestion
  def short_lived?
    return false unless created_at && expires_at
    (expires_at - created_at) < 15.minutes
  end
```

I like the idea of having short lived tokens as a concept and we can add returning the duration later if necessary. I originally put the `(age = 15.minutes)` as an argument, but I don't even think that's necessary for now.

Reply:
I like it, done